src/toil/common.py

```diff
@@ -378,10 +380,15 @@ def _addOptions(addGroupFn, config):
                       "new nodes once the maximum number of preemptable-nodes has been"
                       "reached.")
 
+    addOptionFn('--useLatestNodeTypes', default=False,
```

Comment:
Instance types should always be loaded dynamically.

Reply:
It's required to fetch a 220MiB+ JSON file.  On campus ethernet, that's a couple minute delay, which isn't bad.  But on a slower internet connection, that could be much longer.  If you're very opposed to this, then maybe two other ideas?

1. Just stick with the static lists, and update every release as a PR.  Provide a user command if the user wishes to update Toil's lists internally (this function already exists).

or

2. Serve our own, much smaller, json from a server.  AWS has some kind of notification system when changes occur so that we'd know when to update, though I haven't looked into it much.